tests/python/relax/test_transform_fuse_tir.py

```diff
@@ -1930,5 +1930,251 @@ def main(
     _check(Before, After)
 
 
+def test_inplace_simple():
+    @I.ir_module
+    class Module:
+        I.module_attrs({"foo": "bar"})
+
+        @T.prim_func(private=True)
+        def add_inplace(
+            A: T.Buffer((T.int64(10), T.int64(20)), "float32"), B: T.Buffer((), "float32")
+        ):
+            T.func_attr({"tir.noalias": T.bool(True)})
+            for ax0, ax1 in T.grid(T.int64(10), T.int64(20)):
+                with T.block("T_add"):
+                    v_ax0, v_ax1 = T.axis.remap("SS", [ax0, ax1])
+                    T.reads(A[v_ax0, v_ax1], B[()])
```

Comment:
Nitpick: Unless they're relevant to the specific test case, removing the `T.reads` and `T.writes` annotations can make the test case easier to read.

Reply:
This speaks to my lack of familiarity with TIR :sweat_smile: I'll see how I can compress it

Reply:
No worries, and thank you for the updates!  My preference is to make the test cases have as little extraneous information as possible, even when it requires a few TIR/TVMScript tricks to do so.

I'm wondering if it would be useful to update the TVMScript printing, so the `T.reads` and `T.writes` annotations are only printed if they are different from what would automatically be inferred.  That way, when developers print a function for use in a test case, the duplicate information isn't printed.